docs/user/tutorial/50-news.rst

```diff
@@ -8,6 +8,40 @@ Blog* application.
 Like Aldryn People, Aldryn News & Blog is a separate application that integrates seamlessly with django CMS to provide new functionality for structured information. In this case the structured information is news articles.
 
 
+*********************
+Create an **apphook**
+*********************
+
+Just as we did with Aldryn People, we should create an apphook to create a landing page for the
+news items.
+
+We'll create a *News* page, and add an apphook for the Aldryn News & Blog application to it.
+
+* create and publish a new page called News
+* open page settings
+
+    .. image:: /user/tutorial/images/page-settings-button.png
+        :alt: 'page settings button'
```

Comment:
Don't forget to change alt text as required. It doesn't need to be perfect of course, I can always tweak it before merging.